Typescript compilation fails on v2.1.1+ due to missing jsonwebtoken types

### Describe the problem

Running Typescript compilation on `v2.1.1+` fails with the following error:

```
$ tsc -p tsconfig.json --noEmit
node_modules/jwks-rsa/index.d.ts:3:32 - error TS2307: Cannot find module 'jsonwebtoken' or its corresponding type declarations.

3 import type {Jwt, Secret} from 'jsonwebtoken'
                                 ~~~~~~~~~~~~~~


Found 1 error in node_modules/jwks-rsa/index.d.ts:3
```

We don't use `jsonwebtoken` and instead use `fast-jwt`. Our `package.json` looks like:

```json
{
  "dependencies": {
    "fast-jwt": "1.5.4",
    "jwks-rsa": "2.1.0",
  },
}
```

Downgrading to `v2.1.0` fixes the issue.

### What was the expected behavior?

The package should continue to compile even if we don't have the `jsonwebtoken` package installed.

### Reproduction
Set up a project with the following files:

_package.json_
```json
{
  "name": "jwks-rsa-213-compile-error",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "fast-jwt": "^1.5.4",
    "jwks-rsa": "^2.1.3"
  },
  "devDependencies": {
    "typescript": "^4.7.2"
  }
}
```

_tsconfig.json_
```json
{
    "compilerOptions": {
      "declaration": true,
      "esModuleInterop": true,
      "lib": ["ES2021"],
      "module": "CommonJS",
      "noImplicitAny": true,
      "noImplicitReturns": true,
      "outDir": "dist",
      "resolveJsonModule": true,
      "rootDir": ".",
      "strict": true,
      "target": "ES2021",
      "typeRoots": [
        "./node_modules/@types"
      ]
    },
    "exclude": ["node_modules"],
  }
  
```

_index.ts_
```typescript
import { JwksClient, SigningKey } from 'jwks-rsa';
```

Run the below:
```
$ yarn run tsc -p tsconfig.json --noEmit
yarn run v1.22.18
$ /Users/chingodawita/src/tmp/jwks-rsa-213-compile-error/node_modules/.bin/tsc -p tsconfig.json --noEmit
node_modules/jwks-rsa/index.d.ts:3:32 - error TS2307: Cannot find module 'jsonwebtoken' or its corresponding type declarations.

3 import type {Jwt, Secret} from 'jsonwebtoken'
                                 ~~~~~~~~~~~~~~


Found 1 error in node_modules/jwks-rsa/index.d.ts:3
```

### Environment

- **Version of this library used:** `2.1.3`
- **Which framework are you using, if applicable:** N/A
- **Other modules/plugins/libraries that might be involved:** `fast-jwt`
- **Any other relevant information you think would be useful:** N/A

--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@types/express": "^4.17.13",
+    "@types/jsonwebtoken": "^8.5.8",
     "debug": "^4.3.4",
     "jose": "^2.0.5",
     "limiter": "^1.1.5",
@@ -21,7 +22,6 @@
   "devDependencies": {
     "@types/chai": "^4.2.11",
     "@types/express-jwt": "^6.0.4",
-    "@types/jsonwebtoken": "^8.5.8",
     "@types/mocha": "^5.2.7",
     "@types/nock": "^11.0.0",
     "@types/node": "^14.14.12",

